metadata_writer_sqlite.h

```diff
@@ -65,6 +65,7 @@
                             "NetworkProvider INT," \
                             "NetworkAddressFamily INTEGER NOT NULL," \
                             "NetworkAddress TEXT NOT NULL," \
+                            "Type INTEGER NOT NULL," \
```

Comment:
This is just a question because I am curios. Is the reason we export type that the new importer script needs to know which data it is dealing with? The SQL should not need to contain Type, it is implicit with the table name.